client/src/app/components/discover-import-wizard/useWizardReducer.ts

```diff
@@ -20,17 +24,20 @@ const INITIAL_WIZARD_STATE: WizardState = {
 
 type WizardReducer = (state: WizardState, action: WizardAction) => WizardState;
 type WizardAction =
+  | { type: "SET_PLATFORM"; payload: SourcePlatform | null }
   | { type: "SET_FILTERS"; payload: FilterState }
   | { type: "SET_RESULTS"; payload: ResultsData | null }
   | { type: "RESET" };
 
 const validateWizardState = (state: WizardState): WizardState => {
-  const isReady = state.filters.isValid;
+  const isReady = state.platform !== null && state.filters.isValid;
```

Comment:
just in case I would use a wider condition for the platform ` platform !== null => !!platform`